reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7966,6 +7966,7 @@ public final void subscribe(Subscriber<? super T> actual) {
 	 *      }
 	 *      return flux; //either early complete or error, this forwards the termination in any case
 	 *      //`return flux.onErrorResume(t -> Mono.empty());` instead would suppress an early error
+	 *      //`return Flux.just(1,2,3);` instead would suppress an early error and return 1, 2, 3
```

Comment:
```suggestion
	 *      //`return Flux.just(1,2,3);` instead would suppress an early error and return 1, 2, 3.
	 *      //It would also only cancel the original `flux` at the completion of `just`.
```